CONTRIBUTING.md

```diff
@@ -54,14 +63,15 @@ class FooBarLabelCheck(LabelCheck):
 value_regex=None)
 ```
 
-- `name` means what kind label we check. It has to end with `_label`.
-- `message` is shows in case check is not fulfilled.
-- `description` provides information, why the check has to be specified.
-- `reference_url` references to an URL for specific check.
-- `tags`
-- `label` means name of the label, that will be checked.
-- `required` specifies label has to be present. Otherwise label is not allowed (e.g. deprecated label).
-- `value_regex` means regular expression to check label.
+Let's go through the list of keyword arguments:
+
+- `name` — name of the check, this will be used to reference this check inside a ruleset.
+- `message` — error message when the check fails.
+- `description` gives information in colin's output on what this check does.
+- `reference_url` — URL with even more info about the check. Ideally link to guidelines.
+- `tags` — used to filter checks to run (`colin check -t foo` would run checks which have tag `foo`).
+- `label` is a keyword argument specific to `LabelCheck` class.
+- `required` — global keyword for `Check` class. If the check fails and this is true, the check is marked as failed. If this is false and the check fails, it is a warning.
```

Comment:
This is currently not global -- now, not all checks use that, but maybe it makes sense to make it global.

Reply:
so it's specific for LabelCheck? will update then

Reply:
Hmm, it's messier than I thought:
- InstructionCheck
- DockerfileLabelCheck
- EnvCheck
- LabelCheck